pkg/gui/status_panel.go

```diff
@@ -128,6 +128,52 @@ func (gui *Gui) handleEditConfig() error {
 	return gui.askForConfigFile(gui.helpers.Files.EditFile)
 }
 
+// Minimises a repository url to "author/repo-name" if applicable
+func minimiseURL(url string) string {
+	sshPrefix := "git@github.com:"
+	httpsPrefix := "https://github.com/"
+
+	if s, found := strings.CutPrefix(url, sshPrefix); found {
+		return s
+	}
+
+	if s, found := strings.CutPrefix(url, httpsPrefix); found {
+		return s
+	}
+
+	// do nothing if there is no prefix to remove
+	return url
+}
+
+func (gui *Gui) handleClone() error {
+	return gui.c.Prompt(types.PromptOpts{
+		Title: "Repository url",
+		HandleConfirm: func(url string) error {
+			return gui.c.Prompt(types.PromptOpts{
+				Title: "Destination folder",
+				HandleConfirm: func(destination string) error {
+					message := utils.ResolvePlaceholderString(
+						gui.c.Tr.Cloning,
+						map[string]string{
+							"url":         minimiseURL(url),
+							"destination": destination,
+						},
+					)
+
+					return gui.c.WithLoaderPanel(message, func() error {
+						err := gui.git.Clone.Clone(url, destination)
+						if err != nil {
+							_ = gui.c.Error(err)
+						}
+
+						return gui.c.Refresh(types.RefreshOptions{Mode: types.ASYNC})
```

Comment:
this refresh isn't necessary because we don't actually need to refresh anything.